.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,14 @@
+name: Build and Test
+
+on:
+  push:
+    branches: ['**']
```

Comment:
not needed - you remove this whole line and just have it as `push`

Reply:
add all requested changes in commit 0b1d5327776ed08bb564010ec564045a84dfde2e